doc/source/release/1.24.0-notes.rst

```diff
@@ -168,7 +168,7 @@ Is now always false.  While previously it was true for the specific cast.
 
 Returned arrays respect uniqueness of dtype kwarg objects
 ---------------------------------------------------------
-When the ``dtype`` keyword argument is used with :py:func:`np.array()` or
+When the ``dtype`` keyword argument is used with :py:func:`array()` or
```

Comment:
I don't think this is correct either, python has an `array` module which is independent of numpy. Maybe try `numpy.array`, or just make it a simple ``np.array``. I really don't like links in the release note fragments...

Reply:
> I don't think this is correct either

Good news - this is fine because the `:py:func:` role ensure's it's picking up numpy's array as opposed to the Python built-in array module (you can verify this in the [rendered docs](https://output.circle-artifacts.com/output/job/d59abe6d-c52e-4aab-8215-d546052c1632/artifacts/0/doc/build/html/release/1.24.0-notes.html#returned-arrays-respect-uniqueness-of-dtype-kwarg-objects)